lib/client/jsx/components/list/list-upload-failed.jsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 
-import UploadControl from './upload-control';
+import UploadControl from 'etna/src/jsx/components/upload-control';
```

Comment:
Just a note that these paths changed with the move to `etna` instead of `etna-js`. You may want to update off of the latest `cs-extract-uploader`, once you've pushed your changes to that repo too.

Reply:
Sure.  I'm waiting for your work to hit master both in etna-js and here so I have a better idea of what the final paths are and I'll adjust from there.